src/Microsoft.DotNet.Archive/Microsoft.DotNet.Archive.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>Archive and compression types.</Description>
-    <VersionPrefix>1.0.0-preview4</VersionPrefix>
+    <VersionPrefix>1.0.0-preview5</VersionPrefix>
```

Comment:
does it make sense to put this in one of the props files? (or even the whole property)